inf-clojure.el

```diff
@@ -682,11 +682,24 @@ to continue it."
 (defun inf-clojure-preoutput-filter (str)
   "Preprocess the output STR from interactive commands."
   (inf-clojure--log-string str "<-RES----")
-  (cond
-   ((string-prefix-p "inf-clojure-" (symbol-name (or this-command last-command)))
-    ;; Remove subprompts and prepend a newline to the output string
-    (inf-clojure-chomp (concat "\n" (inf-clojure-remove-subprompts str))))
-   (t str)))
+  (let* ((last-line (car (last (split-string str "\n"))))
+         (response-was-one-line (string= last-line str))
+         (last-line-was-prompt (string-match inf-clojure-prompt last-line))
+         (filtered (cond
```

Comment:
Perhaps `filtered-output` or `processed-output`. `filtered` looks a bit weird to me.